.jenkinsci/docker-cleanup.groovy

```diff
@@ -1,17 +1,14 @@
 #!/usr/bin/env groovy
 
+// remove docker network and stale images
 def doDockerCleanup() {
-
-    sh """
-      # Check whether the image is the last-standing man
-      # i.e., no other tags exist for this image
-      docker rmi \$(docker images --no-trunc --format '{{.Repository}}:{{.Tag}}\\t{{.ID}}' | grep \$(docker images --no-trunc --format '{{.ID}}' ${iC.id}) | head -n -1 | cut -f 1) || true
-      sleep 5
-      docker network rm $IROHA_NETWORK || true
-      #remove folder with iroha.deb package and Dockerfiles
-      rm -rf /tmp/${env.GIT_COMMIT}-${BUILD_NUMBER}
-      rm -rf /tmp/${env.GIT_COMMIT}
-    """
+  sh """
+    # Check whether the image is the last-standing man
+    # i.e., no other tags exist for this image
+    docker rmi \$(docker images --no-trunc --format '{{.Repository}}:{{.Tag}}\\t{{.ID}}' | grep \$(docker images --no-trunc --format '{{.ID}}' ${iC.id}) | head -n -1 | cut -f 1) || true
+    sleep 5
```

Comment:
What is the point of sleep? Can we rework it with the synchronous check of `rmi`?